web-client/src/presenter/actions/validateAddPractitionerAction.ts

```diff
@@ -54,11 +54,11 @@ export const validateAddPractitionerAction = ({
       'birthYear',
       'practitionerType',
       'practiceType',
-      'contact.country',
-      'contact.address1',
-      'contact.city',
-      'contact.state',
-      'contact.postalCode',
+      'country',
+      'address1',
+      'city',
+      'state',
+      'postalCode',
```

Comment:
This was preventing in-line and ErrorNotification messages from displaying correctly.